docs/root/start/install.rst

```diff
@@ -171,6 +168,29 @@ The following commands will pull and show the Envoy version of current images.
          To use the ``nightly`` version from `Get Envoy <https://www.getenvoy.io/>`__
          replace the word ``stable`` with ``nightly`` in the above commands.
 
+   .. tab:: Windows (image)
+
+      You can run Envoy on a Windows host using the official Windows Docker image.
+
+      .. warning::
+
+         This image is currently experimental and includes binaries that are not open source
+
```

Comment:
The first statement is wrong, the port is code complete, and this has been discussed by the maintainers, at the community meeting. Please fix.

The second statement is confusing at best; neither the Windows image nor the OS-free glibc image are exactly satisfied by our Apache License 2.0. A valuable statement here with respect to each docker installation would be a footnote (not "warning") that Windows includes the respective base operating systems (Windows LTSC2019 - long term server core of Server 2019, or alpine-3.12_glibc-2.31, or google vrp context) under their respective licensing. This [link](https://hub.docker.com/_/microsoft-windows-base-os-images) offers the necessary licensing context for the MS Windows docker base image.

Please don't use envoy docs for personal evangelism. Both Windows and Linux docker users locally and users of public clouds are generally aware of what their docker or host licensing prerequisites entail.

Reply:
Reminder that we agreed to drop the statement above in line 177.